js/index.js

```diff
@@ -4,39 +4,100 @@ function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
 
   //... your code goes here
+  const price = +product.querySelector('.price span').innerHTML;
```

Comment:
```suggestion
  const price = +product.querySelector('.price span').innerText;
```
Aqui es mejor usar innerText, siempre que puedas evitarte usar HTML mejor! :D